modules/installation-and-upgrade/pages/server-y.adoc

```diff
@@ -18,6 +18,12 @@ If you are logged into a GNOME session running on the machine you are going to m
 This does not apply if you are logged in from a remote machine (unless you are running a VNC session with GNOME).
 ====
 
+[IMPORTANT]
```

Comment:
I think this could be relevant for `modules/installation-and-upgrade/pages/server-major-upgrade-uyuni.adoc`?

(The .adoc you are changing is only for SUSE Manager).

Can you also add a changelog entry at `.changelog` at the root of the repo? :-)

Reply:
Oh, and clarify if we need to backport this.

I'd say yes, so you need to mark all three checkboxes at the description.

Reply:
I can update .changelog. However there is no hardware requirements for uyuni, this should be created first.

Reply:
We do have them:
https://www.uyuni-project.org/uyuni-docs/en/uyuni/installation-and-upgrade/uyuni-install-requirements.html

So there should be a `uyuni-install-requirements.adoc`, if I am not wrong.

Reply:
I modified the PR accordingly.